dev/reference/tools/br/br-case-demo.md

```diff
@@ -1,80 +1,75 @@
 ---
-title: BR 快速备份恢复最佳实践
-category: how-to
+title: BR 备份恢复案例展示
+category: reference
+aliases: ['/docs-cn/dev/how-to/maintain/backup-and-restore/br-best-practices/']
 ---
 
-# BR 快速备份恢复最佳实践
+# BR 备份恢复案例展示
 
-[Backup & Restore](/dev/reference/tools/br/br.md)（下文简称 BR）是 PingCAP 新推出的分布式快速备份和恢复工具。本文描述了 BR 在备份和恢复场景下的操作过程以及注意事项，供用户参考以达到最佳实践。
+[Backup & Restore](/dev/reference/tools/br/br.md)（下文简称 BR）一款分布式的快速备份和恢复工具。本文展示了四种备份和恢复场景下的 BR 操作过程，以帮助读者达到以下目标：
```

Comment:
- I would want to know what are the four use cases at the beginning. That would help users to know the whole structure and to choose which part they would like to read.
- Add a link at least, or directly show the four use cases at the beginning.

```suggestion
[Backup & Restore](/dev/reference/tools/br/br.md)（下文简称 BR）一款分布式的快速备份和恢复工具。本文展示了[四种备份和恢复场景](#使用场景)下的 BR 操作过程，以帮助读者达到以下目标：
```